util/opentelemetry-util-tracer/src/opentelemetry/util/tracer/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry import trace
+from opentelemetry.trace.span import Span
+from typing import Optional 
+from opentelemetry.util import types
+
+class ServerSpanTracer:
+    "Decorator class for tracer for conditionally creating server spans"
+
+    def __init__(self, tracer, span_name, context, start_time: Optional[int] = None, attributes: types.Attributes = None):
+        self.tracer = tracer
+        self.span_name = span_name
+        self.context = context
+        self.start_time = start_time
+        self.attributes = attributes
+
+    def get_span(self):
+        "Returns a new span with either the current span found as parent or remote span as parent"
```

Comment:
According to PEP 257 (docstring conventions) that I linked above, it should technically be "return" rather than "returns", but it seems like they have written "returns" in other parts of the code... so I think it would be fine by adding the period/triple quotes!
```suggestion
        """Returns a new span with either the current span found as parent or remote span as parent."""
```